### ISSUE TITLE
Setting passcode does not work on Android N

### ISSUE THREAD

**Issue by MisterY** [moneymanagerex/android-money-manager-ex#981]
Crypto provider has been removed in Android N.

Ref:
- http://android-developers.blogspot.co.at/2016/06/security-crypto-provider-deprecated-in.html
- http://stackoverflow.com/questions/36961331/caused-by-java-security-nosuchproviderexception-no-such-provider-crypto-and

**Comment by tschlufter** [comment]
I can't start the app anymore since I installed Android N. 
After I started the app and I entered the passcode it crashes with this error message: "Error: no such provider: crypto".

I'm usinga Nexus 5X with Android 7.0.
The app is in Version 2016.09.02.911.
I downloaded the latest apk from the releases page and installed it but still the same problem.

Anything I can do to get it working again?

Regards
Tobias
